channeldb/graph.go

```diff
@@ -447,6 +453,18 @@ func (c *ChannelGraph) AddChannelEdge(edge *ChannelEdgeInfo) error {
 			return err
 		}
 
+		// Mark edge policies for both sides as unknown. This is to
+		// enable efficient incoming channel lookup for a node.
+		for _, key := range([]*[33]byte {&edge.NodeKey1Bytes, 
+			&edge.NodeKey2Bytes}) {
+			
+			if err := putChanEdgePolicyUnknown(edges, edge.ChannelID, 
```

Comment:
If deprecating the value in the edges map, storing an unknown policy wouldn't be needed anymore.